### ISSUE TITLE
Possible entry in the Troubleshooting Guide related to VPN software on Windows

### ISSUE THREAD

**Issue by gbraad** [minishift/minishift#842]
When using Hyper-V with a virtual switch that is externally connected and a connected VPN solution, such as OpenVPN, might result in unrouted traffic to the same network segment as the assigned IP address. This prevents `minishift` to provisioning the target VM (and/or `minishift ssh` will fail to connect).

Steps to reproduce:
  1. Create a virtual switch for the external network adapter. eg. "External (Wireless)
  2. Extract the minishift release and start the instanceL\:

```cmd
C:\> set HYPERV_VIRTUAL_SWITCH=External (Wireless)
C:\> minishift.exe start
```

     Starting local OpenShift cluster using 'hyperv' hypervisor...
     E0428 15:34:02.215336   10984 start.go:171] Error starting the VM: Error creating the VM. Error creating machine: Error detecting OS: Too many retries waiting for SSH to be available.  Last error: Maximum number of retries (60) exceeded. Retrying.
     E0428 15:43:39.508106   10984 start.go:171] Error starting the VM: Error configuring authorization on host: Too many retries waiting for SSH to be available.  Last error: Maximum number of retries (60) exceeded. 
     Retrying.

Doing `minishift ip` will for instance indicate that the instance actually has been assigned an address (eg. 10.0.21.126). When opening the console and pinging the gateway (`ping 10.0.21.1`), the packets are not routed over the virtual switch, but end up in the VPN. Turning off the VPN will allow the ping and provisioning to succeed.

Proposed solution:
Document this behavior for now...

**Comment by hferentschik** [comment]
Merged via pull request #979

**Comment by gbraad** [comment]
Adding reference to #417 to track behaviour related to Hyper-V networking
